components/edit-collective/EditCollectiveForm.js

```diff
@@ -483,6 +493,21 @@ class EditCollectiveForm extends React.Component {
           type: 'country',
           placeholder: 'Select country',
         },
+        {
+          name: 'currency',
+          type: 'select',
+          defaultValue: get(this.state.collective, 'currency'),
+          options: currencyOptions,
+          when: () => {
+            if (get(this.state.collective, 'isHost')) {
+              return false;
+            }
+            if (get(this.state.collective, 'type') === CollectiveType.COLLECTIVE) {
```

Comment:
No, let's test if it's `User` or `Organization`.

Reply:
added check `collectiveType === User || Organisation` pass